src/styles/stopwatch.scss

```diff
@@ -0,0 +1,73 @@
+.stopwatch {
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
```

Comment:
Not critical in this task, but if you've added these properties just to center the stopwatch better to add some styles to the container above (`body` in this case) to do it